codility_training/lessons.lesson02.Arrays.CyclicRotation/khw970421.js

```diff
@@ -22,6 +22,23 @@ function solution1(A, K) {
     return A;
 }
 
+//방법3
+function solution2(A, K) {
+    const size= A.length;
+    let change = K%A.length;
```

Comment:
```suggestion
    const change = K % A.length;
```
- size 제거
- change의 값이 변하지 않으므로 const로 할당